### PR TITLE
Add displaying excluded and included tests

### PR DESCRIPTION
**Pull request by kstrenkova**

### CODE REVIEW

**Comment by lpiwowar** [comment]
recheck

File: container-images/tcib/base/os/tempest/run_tempest.sh [hunk 1]
```diff
@@ -232,6 +232,16 @@ function run_rpm_tempest {
 function generate_test_results {
     pushd $TEMPEST_DIR
 
+    echo "Excluded tests"
+    if [ ! -z ${TEMPEST_EXCLUDE_LIST} ]; then
+        echo "$(cat ${TEMPEST_EXCLUDE_LIST})"
```
Comment (lpiwowar) [inline container-images/tcib/base/os/tempest/run_tempest.sh:237]: Is the `echo` here really necessary?:)

Comment (kstrenkova) [reply container-images/tcib/base/os/tempest/run_tempest.sh]: Thanks for review, it is not needed. My concern with missing end of line was caused by a small mistake in documentation.

**Review by slagle — APPROVED** [review]

**Pull request merged**